apps/server/test/lexical/server/code_intelligence/completion/env_test.exs

```diff
@@ -365,7 +365,12 @@ defmodule Lexical.Server.CodeIntelligence.Completion.EnvTest do
     end
 
     test "is true if a module reference starts in function arguments" do
-      env = new_env("def my_function(%__|)")
+      env = new_env("def my_function(%_|)")
+      assert in_context?(env, :struct_reference)
+    end
```

Comment:
you need to exercise this more; Think about the ways in which the code you created can fail. For example, you're using a `String.contains?` on the entire line in `env.ex`. That seems really off to me, since you don't seem  to care if the percent comes before _or_ after the cursor.

Reply:
yeah, you're right, I changed to starts_with `percent(%)`

so the case is  `%_.*`, I think we do not need to care about the thing after the cursor.